js/service-worker.js

```diff
@@ -1,4 +1,8 @@
-// import './browser-polyfill.min.js'; // TODO v3 migration
+try {
+  importScripts('./browser-polyfill.min.js'); // v3
```

Comment:
So I guess `importScripts` works in a packaged extension while a regular `import` doesn't?

Reply:
Well, `importScripts` comes with service workers. v3 only allows one entry point for a background script(worker), but I needed to somehow add a polyfill. This actually can be either `import` or `importScripts` in v3, but `import` statement throws a nasty error that I can not catch in v2: `Uncaught SyntaxError: Cannot use import statement outside a module`

So neither `importScripts` nor `import`'s are supported in v2. But we don't really need them there, since we can add multiple background scripts to the manifest. Fortunately the `importScripts` error is catchable. 

Hope this makes sense, I know it's a real mess and a bit confusing.

Reply:
Yep no worries, I think a certain amount of mess is just the way it goes with these extensions, especially with all the permissions and manifest changes.